### PR TITLE
Add protection in use

### PR DESCRIPTION
**Pull request by dcmiddle**
Replace "encryption" to "cryptographic protection" to more accurately cover confidentiality and integrity.

Add techniques for protecting data during execution to complete the triad of in transit, at rest, and in use.

cleanup secretions.

### CODE REVIEW

**Review by lehors — APPROVED** [review]
I like it. Thanks @dcmiddle !

**Review by SecurityCRob — APPROVED** [review]
Excellent addition @dcmiddle , tyvm

File: process/security_baseline.md [hunk 1]
```diff
@@ -231,6 +234,26 @@ Encrypting data in transit ensures that data transmitted over networks or betwee
   * Ed25519:
     * Fixed key length: 256 bits
 
+#### Data in Use
```
Comment (steiza) [inline process/security_baseline.md:237]: The content in this section is true, but I want to clarify the intention - is this section informational, or are we requiring TIs make use of confidential computing as part of security baseline / lifecycle maturation?

Providing this information to TIs is fine. I don't think we should require TIs make use of confidential computing in their lifecycle maturation.

Comment (dcmiddle) [reply process/security_baseline.md:237]: Thanks for the feedback. Can you say at what maturity level you think projects should protect data in use?

Comment (steiza) [reply process/security_baseline.md:237]: Well there's the capability ("protect data in use") and ways to implement that capability ("use confidential computing"). The way the document is structured, it seems like we're requiring TIs to use confidential computing in order to be graduated, which I don't agree with.

Maybe we could move this section to the end of the doc under its own heading, like "Informational" or "Additional Security Steps"?

Comment (steiza) [reply process/security_baseline.md:237]: Oops! I didn't realize the preview was collapsed, and this section is already in "Appendix" - never mind!

**Review by steiza — APPROVED** [review]

**Review by bobcallaway — APPROVED** [review]

**Pull request merged**